src/Stripe/Infrastructure/ParameterBuilder.cs

```diff
@@ -77,6 +77,16 @@ public static string ApplyAllParameters(this StripeService service, object obj,
                             var stripeAccountBankAccountOptions = (StripeAccountBankAccountOptions)value;
                             newUrl = ApplyNestedObjectProperties(newUrl, stripeAccountBankAccountOptions);
                         }
+                        else if (property.PropertyType == typeof(StripeAccountLegalEntityOptions))
+                        {
+                            var sripeAccountLegalEntityOptions = (StripeAccountLegalEntityOptions)value;
+                            newUrl = ApplyNestedObjectProperties(newUrl, sripeAccountLegalEntityOptions);
+                        }
+                        else if (property.PropertyType == typeof(StripeAccountTosAcceptanceOptions))
+                        {
+                            var stripeAccountTosAcceptanceOptions = (StripeAccountTosAcceptanceOptions)value;
+                            newUrl = ApplyNestedObjectProperties(newUrl, stripeAccountTosAcceptanceOptions);
+                        }
```

Comment:
you did this correctly, but I need to make a separate class with a list of types or something instead of this madness. adding this note for myself. :)